promscale/installation/index.md

```diff
@@ -30,36 +25,37 @@ and adds its public schemas to the search path. This makes querying Promscale
 data easier. The public schemas that Promscale adds are: `ps_tag`, `prom_api`,
 `prom_metric`, `ps_trace`. 
 </highlight>
+ 
 
 ## Install Promscale without instrumentation
-If you have Prometheus or OpenTelemetry installed, you can install Promscale
-on these environments:
-
-*   **Kubernetes**:  use [Helm charts][promscale-install-helm] or [a manifest][promscale-install-k8s-manifest] file.
-*   **Docker**: use a [pre-built Docker container][promscale-install-docker].
-*   **VM or bare metal**: use the [source][promscale-install-source] file.
+If you want to install Promscale on your infrastructure where you have already installed Prometheus or OpenTelemetry, you can do that on:
```

Comment:
This sounds like Prometheus or OpenTelemetry should be in the same infra as where Promscale is about to be installed. Promscale can connect to both Prometheus and OpenTelemtry anywhere outside the machines/cluster it is running on. 

The former definition sounds more simple.